mag/parser.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform MAG Online
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  // let param = parsedUrl.query || {};
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+
+  let match;
+
+  if ((match = /^\/doi\/epdf\/(([0-9.]+)\/([a-z0-9.]+))$/i.exec(path)) !== null) {
+    // https://www.magonlinelibrary.com/doi/epdf/10.12968/hmed.2020.0321
+    result.rtype    = 'ARTICLE';
+    result.mime     = 'PDF';
+    result.doi = match[1];
+    result.unitid = match[1];
+
+  } else if ((match = /^\/doi\/epdfplus\/(([0-9.]+)\/([a-z0-9.]+))$/i.exec(path)) !== null) {
+    // https://www.magonlinelibrary.com/doi/epdfplus/10.12968/hmed.2020.0321
+    result.rtype    = 'ARTICLE';
+    result.mime     = 'PDFPLUS';
```

Comment:
In counter 5, PDFPLUS is considered as a PDF. We will remove this mime soon. You need to change to PDF